index.html

```diff
@@ -5,22 +5,58 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
+		<title>Relaxr</title>
+		<!-- Google fonts -->
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/normalize.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
```

Comment:
@chrislindor Excellent job at sectioning off your code into manageable portions (header, main, footer).

:hammer: I also see that you (correctly) used the container to house all content on the page. Why not use it to your advantage to get the content to be 960px wide like the diagram shows? You can use the margin trick to center it as well. Do you remember it? It will **auto**matically adjust the left and right margins to be equal values.